tools/issue-labeler/src/Azure.Sdk.Labels/Azure.Sdk.LabelTrainer.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
```

Comment:
This project is just a command-line app that trains models.   Shouldn't have a need for integration with Azure Functions.